### ISSUE TITLE
Feature request: Black background

### ISSUE THREAD

**Issue by hap2001gh** [memtest86plus/memtest86plus#439]
The current blue background strain user's eye a lot when used at night, especially with screens that cannot be adjusted (such as laptops).
Debian CLI installer have such option to make background black, I hope we can also have that.

**Comment by x86fr** [comment]
That's a feedback I already got from betatesters. I'm working on a PR to customize background color

**Comment by debrouxl** [comment]
This has already come up in e.g. #323 as well.
Do you plan on implementing a simple customization of the background color, or instead a couple presets, the current one and another one for dark mode ? The latter would be more coherent and flexible, at least,
In very old BIOS implementations, I've already seen some color video testing modes (I fondly remember about an orange - green - magenta - white color set), but no need to go _that_ overboard nowadays ^^

**Comment by hap2001gh** [comment]
I think a single black mode is good enough, I just found memtest86.com version already using that.

**Comment by 01e3** [comment]
Hi @x86fr, have you maybe started working on this? If not, I have some ideas how this can be done and I would be happy to spend some time on this in the following days.

**Comment by x86fr** [comment]
Yes. I wrote a patch about 6 months ago and it's deployed since then on a testing row of the SI factory. I'll try to convert it to a PR next week.
